src/pip/_internal/cli/cmdoptions.py

```diff
@@ -774,6 +775,13 @@ def _handle_no_use_pep517(
         """
         raise_option_error(parser, option=option, msg=msg)
 
+    # If user doesn't wish to use pep517, we check if setuptools is installed
+    # and raise error if it is not.
+    if not importlib.util.find_spec("setuptools"):
+        msg = """It is not possible to use --no-use-pep517 without setuptools
+        installed."""
```

Comment:
Perhaps we could give some guidance? Something like:

> It is not possible to use --no-use-pep517 without setuptools
> installed. Either install setuptools or drop this option.

Reply:
IIRC pip generally does not hard-wrap messages, so this should be one line.

Reply:
Oh `raise_option_error` reflows this anyway, nevermind.

Reply:
@hroncok since pyproject.py already has logic for handling some invalid uses of `--no-use-pep517`, it might be better add this one there too, to keep everything in the same place.

Reply:
I would still prefer this not be wrapped. On seeing it, my immediate thought was that the wrapping is at an odd place, and having to search to determine it doesn't actually matter is an annoyance.